### ISSUE TITLE
Question : OnDisconnect not fixed yet ?

### ISSUE THREAD

**Issue by josef256** [Polidea/react-native-ble-plx#697]
the onDisconnect Listener is fixed ? cause the branch which was supposed to fix it was merged but after updating the library the issue still persist.

tested on Android.
#679

**Comment by iDorosh** [comment]
I am also still having the same issue on android.

**Comment by iDorosh** [comment]
Curious about this too. Disconnect listener not working on android

**Comment by Barbiero** [comment]
while #679 was definitely merged, the patch itself was not released on npm nor github for that matter - latest is still 2.0.0.

As a workaround, until it releases, you can install the package by the most recent commit:

```
npm install https://github.com/Polidea/react-native-ble-plx.git#387e4f080f953b0476faa9c06cdd448aa7a1a1b4
```

**Comment by iDorosh** [comment]
Ok perfect thanks for the info! Works great

**Comment by dariuszseweryn** [comment]
Just released `2.0.1`
